### ISSUE TITLE
Improve the UI of landing page.

### ISSUE THREAD

**Issue by Hemanth-Asp** [imhardikdesai/Quiz-App#7]
I would like to update the UIof landing page improve the css to it.Would love to contribute for it

**Comment by imhardikdesai** [comment]
@Hemanth-Asp sure, please start working on it. If you have any ideas then share it with me 😉

**Comment by Hemanth-Asp** [comment]
![UI](https://github.com/imhardikdesai/Quiz-App/assets/119241921/6ba1b5d6-d850-4596-a9dd-c2c1c6fa2c50)
Here is a fast ui developed please suggest changes then i will work on it

**Comment by imhardikdesai** [comment]
@Hemanth-Asp looks nice but please make Minor changes
- Remove underline in Header
- make start button full width
- proper alignment of number of questions
- make sure it will looks good in mobile screen 
- instead of question mark ❓ image you can use Lottie animations (optional)

**Comment by Hemanth-Asp** [comment]
Yeah sure @imhardikdesai will make the changes accordingly and start the coding part

**Comment by rishi457** [comment]
I would like to update the UI of landing page improve the css to it.Would love to contribute for it

**Comment by rishi457** [comment]
![Screenshot (162)](https://github.com/imhardikdesai/Quiz-App/assets/69967446/4ec3848c-8d67-4c2b-89d3-42c790e99bde)
I have submitted a merge request aimed at enhancing the landing page's UI to elevate its professional appeal and user experience.
